includes/helper/wildcardHandler.py

```diff
@@ -37,6 +37,12 @@ def replaceWildcards(text, data):
 		text = text.replace("%BR%", "\r\n")
 		text = text.replace("%LPAR%", "(")
 		text = text.replace("%RPAR%", ")")
+		text = text.replace("%BTA%", "<b>")
+		text = text.replace("%BTE%", "</b>")
+		text = text.replace("%ITA%", "<i>")
+		text = text.replace("%ITE%", "</i>")
+		text = text.replace("%UTA%", "<u>")
+		text = text.replace("%UTE%", "</u>")
```

Comment:
Müssen die denn als Wildcard gesetzt werden?
Also gab es Probleme die einfach direkt in den MSG String zu schreiben?

Reply:
Habe ich ehrlich gesagt garnicht getestet. Dachte mir als Wildcard ist das auch für die Leute einfach die mit der HTML Textfomatierung nichts anfangen können.

Reply:
Ehrlich gesagt finde ich die HTML Tags deutlich einfacher als per Wildcard :-D
N paar Grundlegende HTML Tags sollte jeder kennen, der so eine Software in Betrieb nehmen kann.

Und selbst wenn, sollte man die Wildcards wenn dann direkt im Plugin ersetzen und nicht im globalen Wildcard Handler. Denn die Wildcards funktionieren ja auch nur für das eine Plugin und nicht bei jedem.

Reply:
@N3R6E wurde hier noch etwas angepasst?